packages/realtime_client/lib/src/realtime_client.dart

```diff
@@ -353,7 +358,7 @@ class RealtimeClient {
 
     for (final channel in channels) {
       if (token != null) {
-        channel.updateJoinPayload({'user_token': token});
```

Comment:
The realtime-js library uses `access-token` [see here](https://github.com/supabase/realtime-js/blob/95ab09e47ed7732eb21ac31eac743704bacc29aa/src/RealtimeClient.ts#L322) instead of `user_token`. And I guess that's the correct payload. It got changed to access_token [here](https://github.com/supabase/realtime-js/commit/b3eafe820f61afdb4c663fa600c4f209757ba8b4#diff-f5a6b192eb9d6c05f61c33d8629b814ad893fbfb729bfa7af9b226e5c87d2e85)